content/blog/2024/06/2024-06-13-thanks-mirror-sponsors.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Thank you supporters of Jenkins providing download Mirrors"
+tags:
+- community
+- infrastructure
+- sponsors
+- mirrors
+- supporters
+author: dduportal
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+description: >
+  We are grateful for all the support from our supporters providing mirrors for Jenkins downloads.
+---
+
+The Jenkins project artefacts are provided through download mirror servers provided by organization supporting the project.
+
+You can see the list from the link:https://get.jenkins.io/index.html?mirrorstats[mirror status page].
+
+We're really grateful for all the support provided by the organization behind the curtain, namely:
+
+- link:https://osuosl.org[The Oregon State University Open Source Lab] in USA
+- link:https://www.tsinghua.edu.cn[The Tsinghua University] in China
+- link:https://xmission.com[XMission] in USA
+- link:https://belnet.be/[Belnet] in Belgium
+- link:https://www.rwth-aachen.de/[The RWTH Aachen University] in Germany
+- link:https://www.yamagata-u.ac.jp/en/[The Yamagata University] in Japan
+- link:http://freedif.org/[FreeDiff] in Singapore
+- link:https://servanamanaged.com/[Servana] in England
```

Comment:
When I zoom very close into the Singapore area of the mirror list, I see Servana in Singapore, very near FreeDif
```suggestion
- link:https://servanamanaged.com/[Servana] in Singapore
```